ers-protobuf-dbwriter/dbwriter.py

```diff
@@ -157,33 +158,23 @@ def process_issue( issue, session, cursor ) :
     # heavy information
     add_entry("inheritance", '/'.join(issue.inheritance), fields, values)
     add_entry("message", issue.message, fields, values)
-    add_entry("params", convert_params(issue.parameters), fields, values)
-    
-
-    command = "INSERT INTO " + table_name;
-    command += " (" + ", ".join(fields) + ')'
-    command += " VALUES " + repr(tuple(values)) + ';'
+    add_entry("params", issue.parameters, fields, values)
 
+    command = f'INSERT INTO {table_name} ({",".join(fields)}) VALUES ({("%s, " * len(values))[:-2]});'
     logging.debug(command)
```

Comment:
I think you also want to escape field names here. Also why do you need the `[:-2]`?

Reply:
escape the fields, and the table_name

Reply:
the `[:-2]` is to remove the last comma and the last ` ` from the string. I'll escape the rest too.